### PR TITLE
Add box to display value returned and add input box for commands

### PR DESCRIPTION
**Pull request by Ethan-Bradley-Bluefruit**

### CODE REVIEW

File: mock_arduino.py [hunk 1]
```diff
@@ -1,2 +1,10 @@
-def mock_arduino_response():
-    print("Your command did something!")
+from typing import Text
```
Comment (jack-skerrett-bluefruit) [inline mock_arduino.py]: I'd delete this import you don't need, then delete the empty lines below

File: mock_arduino.py [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+
+
+def mock_arduino_response(text):
+    if text == "ledon":
+        return str("ledon")
```
Comment (jack-skerrett-bluefruit) [inline mock_arduino.py]: You don't need to convert these strings to strings, they already be strings :D (if the conversion needs to happen for the reply from a real arduino, that will happen in the serial file, so we don't need to worry about it here).

File: serial_gui.py [hunk 3]
```diff
@@ -1,10 +1,26 @@
 import tkinter
+from tkinter.constants import END
+from typing import Text
```
Comment (jack-skerrett-bluefruit) [inline serial_gui.py]: these imports arent used, they can be deleted

File: serial_gui.py [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
 from mock_arduino import mock_arduino_response
 
 main_window = tkinter.Tk()
 
-serial_connection_button = tkinter.Button(main_window, text = "Hello", command = mock_arduino_response)
-serial_connection_button.pack()
-main_window.mainloop()
+def send_button_click():
+    output_box.configure(text=mock_arduino_response(input_box.get("1.0", 'end-1c')))
+    print(input_box.get("1.0", 'end-1c'))
```
Comment (jack-skerrett-bluefruit) [inline serial_gui.py:10]: this print line seems to act as some debug, so can be removed

**Review by jack-skerrett-bluefruit — CHANGES_REQUESTED** [review]

File: mock_arduino.py (hunk 1 above)
Comment (Ethan-Bradley-Bluefruit) [reply mock_arduino.py]: Done :)

File: mock_arduino.py (hunk 2 above)
Comment (Ethan-Bradley-Bluefruit) [reply mock_arduino.py]: Nice! done

File: serial_gui.py (hunk 3 above)
Comment (Ethan-Bradley-Bluefruit) [reply serial_gui.py]: removed

File: serial_gui.py (hunk 4 above)
Comment (Ethan-Bradley-Bluefruit) [reply serial_gui.py:10]: I tried removing this and it broke, so it's staying in :D

**Review by jack-skerrett-bluefruit — APPROVED** [review]

**Pull request merged**